factory/config.example.yaml

```
@@ -0,0 +1,22 @@
+## SPDX-FileCopyrightText: 2023 Open Networking Foundation <info@opennetworking.org>
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+info:
+  version: 1.0.0
+  description: UDR initial local configuration
+
+configuration:
+  sbi: # Service-based interface information
```

As per its abbreviation in 23.501:
```suggestion
  sbi: # Service Based Interface
```